doc/source/api/general/case_reader.rst

```diff
@@ -50,3 +50,11 @@ Along with basic functionality, the case reader provides many additional feature
     >>> reader.rp_vars()
     >>> reader.config_vars()
 
+
+.. automodule:: ansys.fluent.core.filereader.case
+   :members:
+   :show-inheritance:
+   :undoc-members:
```

Comment:
Should we really be setting this?  On the other hand why would there be undocumented members?